CHANGELOG.md

```diff
@@ -297,13 +345,6 @@
 ## [0.1.20](https://github.com/kubeflow/pipelines/tree/0.1.20) (2019-05-14)
 [Full Changelog](https://github.com/kubeflow/pipelines/compare/0.1.19...0.1.20)
 
- **Closed issues:**
-
```

Comment:
Maybe don't delete these? I noticed this problem last week as well, and manually copied the lines from the new release over, leaving previous release changelogs intact.

Reply:
ah, i didn't notice that it removes things. corrected it in the new commit.